config/app/com/microsoft/azure/iotsolutions/uiconfig/services/external/PackageValidation/PackageValidator.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+package com.microsoft.azure.iotsolutions.uiconfig.services.external.PackageValidation;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import com.microsoft.azure.iotsolutions.uiconfig.services.exceptions.InvalidInputException;
+import play.libs.Json;
+
+public abstract class PackageValidator implements IPackageValidator {
+
+
+    @Override
+    public JsonNode getPackageContent(String pckg) throws InvalidInputException {
+        try
+        {
```

Comment:
Nit: { goes on previous line

Reply:
Yes as discussed we will do this in different PR.